src/guides/v2.3/frontend-dev-guide/layouts/xml-instructions.md

```diff
@@ -61,12 +61,13 @@ We recommend always adding a `name` to blocks. Otherwise, it is given a random n
 | Attribute | Description | Values | Required? |
 |:------- |:------ |:------ |:------ |
 | `class` | Name of a class that implements rendering of a particular block. An object of this class is responsible for actual rendering of block output. | A fully-qualified class name, such as `Vendor\Module\Block\Class`. Defaults to `Magento\Framework\View\Element\Template`. | no |
-|`name` | Name that can be used to address the block to which this attribute is assigned. The name must be unique per generated page. If not specified, an automatic name will be assigned in the format <code>ANONYMOUS_<em>n</em></code> | 0-9, A-Z, a-z, underscore (_), period (.), dash (-). Should start with a letter. Case-sensitive. | no |
+| `display` | Prevents a block from displaying (the associated PHP classes are still loaded). | `true` or `false` | `false` |
```

Comment:
The last column of this table is about the "Required" attribute property. So, it's supposed to be yes or no.